templates/estafette-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!{{.Shell}}
+set -e
+
+{{- range .Commands}}
+echo -e "\x1b[38;5;250m> {{.}}\x1b[0m"
+{{.}} &
```

Comment:
It turns out that variables set in a background process aren't visible for the other commands; so still have to detect when a command is a variable assignment and run it in the foreground.